cards_main.py

```diff
@@ -194,6 +213,9 @@ def playTrick(self, start):
 		if self.trickNum == 0:
 			startPlayer = self.players[start]
 			addCard = startPlayer.play(option="play", c='2c')
+			print(type(addCard))
+			print("why ?????")
```

Comment:
```suggestion
			print("why ?????")
```
Seems like a no purpose line, advice removing it